frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -178,6 +177,7 @@ export default function AnalysisParamForm({
   pulse3dVersions,
   resetDropDown,
 }) {
+  const [disableAxisIsChecked, setDisableAxisIsChecked] = useState(false);
```

Comment:
I know from this PR that this means the y-axis, but someone reading this code will just see `axis` and not know which one. I think this should be named something like `disableYAxisNormalization`